draft-ietf-tls-tls13.md

```diff
@@ -2301,9 +2303,14 @@ EdDSA algorithms
 : Indicates a signature algorithm using EdDSA as defined in
   {{RFC8032}} or its successors. Note that these correspond to the
   "PureEdDSA" algorithms and not the "prehash" variants.
+
+ecdsa_sha1
```

Comment:
That is both ecdsa_sha1 and rsa_pkcs1_sha1 have the same property regarding their use.  We don't like them, but we have to concede that they might appear in certificates.